src/Raven.Client/Documents/Session/AbstractDocumentQuery.cs

```diff
@@ -1134,6 +1146,26 @@ private void BuildFrom(StringBuilder writer)
             FromToken.WriteTo(writer);
         }
 
+        private void BuildDeclare(StringBuilder writer)
+        {
+            DeclareToken?.WriteTo(writer);
```

Comment:
Is there ever only a single `DeclareToken` ? I think that there could be multiple of them.